ironfish/src/wallet/wallet.ts

```diff
@@ -397,6 +387,45 @@ export class Wallet {
     return decryptedNotes
   }
 
+  async connectBlock(
```

Comment:
later this should only update accounts where header.previousHash === account.head